src/routes/scout/ActionInputs.svelte

```diff
@@ -29,133 +29,147 @@
 		actionState = `ScoreBalloon${where}`;
 	}
 	function complete(success: boolean) {
-		// Assume that the robot ejects even if they fail to score
-		if (actionState.includes('IntakeBalloon')) held_balloons++;
-		else if (actionState.includes('IntakeBunny')) held_bunnies++;
-		else if (actionState.includes('IntakeTote')) held_totes++;
-		else if (actionState.includes('ScoreBalloon')) held_balloons--;
+		// Assume that a failure means a note remains where it is
+		if (success) {
+			if (actionState.includes('IntakeBalloon')) held_balloons++;
+			else if (actionState.includes('IntakeBunny')) held_bunnies++;
+			else if (actionState.includes('IntakeTote')) held_totes++;
+			else if (actionState.includes('EjectBalloon')) held_balloons--;
+			else if (actionState.includes('EjectBunny')) held_bunnies--;
+			else if (actionState.includes('EjectTote')) held_totes--;
+		}
+		// Assume failed scoring is still ejecting
+		if (actionState.includes('ScoreBalloon')) held_balloons--;
 		else if (actionState.includes('ScoreBunny')) held_bunnies--;
-		else if (actionState.includes('EjectBalloon')) held_balloons--;
-		else if (actionState.includes('EjectBunny')) held_bunnies--;
-		else if (actionState.includes('EjectTote')) held_totes--;
 
 		const action: AutoActionData = {
 			action: actionState as AutoAction,
 			success: success
 		};
-		actions.push(action);
+		actions.unshift(action); // TODO: Make sure unshift works
 		actionState = 'None';
 	}
 
-	const is_none_state = $derived(actionState === 'None');
-	const is_intake_state = $derived(actionState === 'Intake');
-	const is_score_state = $derived(actionState === 'Score');
+	const is_none_state: boolean = $derived(actionState === 'None');
+	const is_intake_state: boolean = $derived(actionState === 'Intake');
+	const is_score_state: boolean = $derived(actionState === 'Score');
 	const is_eject_state = $derived(actionState === 'Eject');
 </script>
 
-<h1>Number of pieces currently held: {held_scorables}</h1>
-<div class="grid gap-2 grid-cols-1 grid-rows-2 place-items-center">
+<div class="align-center flex min-w-[60%] flex-grow flex-col">
 	{#if is_none_state}
-		<div class="grid gap-2 grid-cols-2">
-			<button class="bg-zinc-500 p-2 rounded" onclick={intake_piece}>Intake</button>
+		<div
+			class="-ml-12 -mr-12 mb-2 mt-2 grid flex-grow grid-cols-1 grid-rows-3 gap-2 text-2xl font-bold"
```

Comment:
way too much negative values, should instead set max-w min-w and w on parent div

Reply:
also could be using mx instead of ml and mr